habs_portal/static/js/partials/StationMetadata.html

```diff
@@ -0,0 +1,14 @@
+<% collection.each(function(model) { %>
```

Comment:
this is how I do it as well. Not sure if it's backbone-ish but it works and it's very understandable.